modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -667,7 +678,8 @@ export class Wallet implements IWallet {
    * @param {Number} params.limit                for routeName === 'consolidate'
    *                 params.maxNumInputsToUse    for routeName === 'fanout'
    *                  - maximum number of unspents you want to use in the transaction
-   * @param {Number} params.numUnspentsToMake - the number of new unspents to make
+   * @param {Number} params.numUnspentsToMake - the number of new unspents to make. It is not applicable for if bulk consolidate.
+   * @param {Boolean} params.bulk - if true, consolidate unspents in multiple transactions composed of 200 unspents each
```

Comment:
Question: each transaction is not necessarily 200 inputs in bulk, right? Maybe a more correct comment is `composed with a maximum of 200 unspents per transaction`